src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookConnector.java

```diff
@@ -41,22 +45,41 @@ public WebhookConnector(String projectKey, String webhookUrl, String webhookSecr
 			this.rootTypes = rootTypes;
 		}
 		this.elementIds = new ArrayList<Long>();
+
+		this.receiver  = "Other";
+		if (this.url != null){
+			if(url.matches("https://hooks.slack.com(\\S*)")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)